app/views/spree/api/images/update_product_image.v1.rabl

```diff
@@ -0,0 +1,5 @@
+object @image
+attributes(*image_attributes)
+attributes :viewable_type, :viewable_id
+node( :thumb_url ) { @product.images.first.attachment.url(:mini) }
+node( :image_url ) { @product.images.first.attachment.url(:product) }
```

Comment:
what about products without image?

Reply:
This just renders the API response after a new image is uploaded, right? It won't be used for products without images.

Reply:
Is there any error handling or validation to enforce that? I'm not sure a nil would not slip through in the case of image creation.

Reply:
The uploading and validation is handled by `angularjs-file-upload`.